test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -457,28 +457,20 @@ describe("UserWallet - Lightning Pay", () => {
         const { BTC: intermediateBalance } = await userWallet1.getBalances()
         expect(intermediateBalance).toBe(initBalance1 - amountInvoice * (1 + initialFee))
 
-        await waitFor(async () => {
-          try {
-            await cancelHodlInvoice({ id, lnd: lndOutside1 })
-            return true
-          } catch (error) {
-            baseLogger.warn({ error }, "cancelHodlInvoice failed. trying again.")
-            return false
-          }
-        })
+        await cancelHodlInvoice({ id, lnd: lndOutside1 })
```

Comment:
also this needs to be in a waitfor (cancelHodlInvoice sometimes throws connection issues, maybe related to test environment or to slow environments)